relayer-cli/src/commands/create/connection.rs

```diff
@@ -17,37 +17,46 @@ use crate::prelude::*;
 #[derive(Clone, Command, Debug, Parser)]
 pub struct CreateConnectionCommand {
     #[clap(
+        long = "a-chain",
         required = true,
+        value_name = "A_CHAIN_ID",
         help = "identifier of the side `a` chain for the new connection"
     )]
     chain_a_id: ChainId,
```

Comment:
Maybe it's best to also rename the struct members to match the option names? Otherwise we'd be always confused.

Reply:
Let's keep the fields as-is, this PR has already gone through a lot of churn and I would rather merge it sooner than later. Let's perhaps do this if there's a follow-up PR (eg. in the one for unit tests)